src/app/lightning/update-ln-payments.ts

```diff
@@ -12,63 +12,80 @@ export const updateLnPayments = async (): Promise<true | ApplicationError> => {
 
   const lndService = LndService()
   if (lndService instanceof Error) return lndService
+
+  const listFns = [lndService.listSettledPayments, lndService.listFailedPayments]
   const pubkeys = lndService
     .listActivePubkeys()
     .filter((pubkey) => pubkeysFromPayments.has(pubkey))
 
-  for (const key of pubkeys) {
-    const pubkey = key as Pubkey
+  for (const listFn of listFns) {
+    for (const key of pubkeys) {
+      processedLnPaymentsHashes = await updateLnPaymentsByFunction({
+        processedLnPaymentsHashes,
+        incompleteLnPayments,
+        pubkey: key as Pubkey,
```

Comment:
the type assignment should not be necessary. if `lndService.listActivePubkeys()` is not typing the result to `Pubkey`, this should be added there, not here